src/@config/colors.ts

```diff
@@ -1,3 +1,33 @@
+// * 수정중...
+
+export const COLOR = {
+  WHITE: "#FFFFFF",
+  PAPER: "#F6F6F6",
+  BLACK: "#000000",
+  BLUE: "#03336a",
+  GREY_LIGHT: "#f0f0f0",
+  GREY: "#808080",
+  GREY_CLEAR: "#d4d4d4",
+  BLACK_OFF: "#444444",
+  TAN: "#d2b48c",
+  RED_LIGHT: "#ff9999",
+  RED: "#ff0000",
+  LEMON: "#FEECB7",
+  MINT_LIGHT: "#ecfad7",
+  MINT: "#E0F3C2",
+  STRAWBERRY_LIGHT: "#ffebeb",
+  STRAWBERRY: "#FF9999",
+  LAVENDER_LIGHT: "#e8ecfa",
+  LAVENDER: "#BAC7F7",
+  LAVENDAR_DARK: "#4b6ff2",
+};
+
+export type ColorKeys = typeof COLOR extends Record<infer U, string>
+  ? U
+  : never;
+
+export type ColorValue<K extends ColorKeys> = typeof COLOR[K];
```

Comment:
```suggestion
export type ColorKeys = typeof COLOR extends Record<infer U, string>
  ? U
  : never;

export type ColorValues<K extends ColorKeys> = typeof COLOR[K];
```

COLOR 객체의 key 값이 담긴 Union 타입이 Keys니까 값들이 담긴 타입은 Values로 통일되어야 할 것 같아요~

Reply:
쓰지 않는 부분인데 정리가 덜 됐네요🥲
COLOR 상수 적용으로 통일하는 작업을 이제 시작해서.. 정리하고 커밋해놓겠습니다~